integration-tests/utils.ts

```diff
@@ -7,8 +7,19 @@ import * as AWS from 'aws-sdk';
 import axios from 'axios';
 import { Chance } from 'chance';
 
-export const getFhirClient = async () => {
-    const { API_URL, API_KEY, API_AWS_REGION, COGNITO_USERNAME, COGNITO_PASSWORD, COGNITO_CLIENT_ID } = process.env;
+export const getFhirClient = async (
+    isAuditor: boolean = false,
+    providedAccessToken: string | undefined = undefined,
```

Comment:
Can we use strings for the role instead? Otherwise it's hard to understand the meaning of something like `getFhirClient(false)`
Also, `providedAccessToken` is effectively an optional parameter
```suggestion
    role: 'auditor'|'practitioner' = 'auditor',
    providedAccessToken: string?
```

Reply:
updated